homeassistant/components/recorder/__init__.py

```diff
@@ -182,6 +187,16 @@
 )
 
 
+# Pool size must accommodate Recorder thread + All db executors
+MAX_DB_EXECUTOR_WORKERS = POOL_SIZE - 1
+
+
+@bind_hass
```

Comment:
Why do we need to decorate this with `@bind_hass`?

Reply:
Historically anything that was expected to be called from another component seems to be decorated with this.  I think we are moving away from this pattern as it's not good for typing.

Reply:
Correct. Let's not introduce new `bind_hass` (and one day kill them all)

Reply:
Removed